sartools/test-data/test_output_target.html

```diff
@@ -1,5 +1,5 @@
 label	files	group
-group1_rep1	dataset_1.dat	group1
-group1_rep2	dataset_2.dat	group1
-group2_rep1	dataset_3.dat	group2
-group2_rep2	dataset_4.dat	group2
+group1_rep1	dataset_[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}.dat	group1
+group1_rep2	dataset_[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}.dat	group1
+group2_rep1	dataset_[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}.dat	group2
+group2_rep2	dataset_[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}.dat	group2
```

Comment:
Adjusted test to take into account Galaxy change in naming dataset files (UUID instead of ID).